profile/README.md

```diff
@@ -5,7 +5,8 @@
 <!--
 TODO: 이미지 변경하기
 -->
-![An illustration showing a variety of differently themed Octocats. Monuments from different cities are indicated in the background like the Space Needle, Berlin Fernsehturm and Transamerica Pyramid.](https://user-images.githubusercontent.com/3369400/133268513-5bfe2f93-4402-42c9-a403-81c9e86934b6.jpeg)
+![Welcome Image with PLAIF members](https://user-images.githubusercontent.com/3369400/133268513-5bfe2f93-4402-42c9-a403-81c9e86934b6.jpeg)
+![Welcome Image with PLAIF members](https://raw.githubusercontent.com/plaif-dev/.github/main/assets/main/opening.png)
```

Comment:
Cool! What would be good between these two?

1. `https://raw.githubusercontent.com`/plaif-dev/.github/main/assets/main/opening.png
2. `https://github.com`/plaif-dev/.github/main/assets/main/opening.png

I am asking this since I found the first one is de facto standard way to add image file

Reply:
> I am asking this since I found the first one is de facto standard way to add image file

I believe that method is used when a file is remotely accessed.

How about using relative link is a tidier in this one since `assets` folder exists. I propose this one
```diff
- ![Welcome Image with PLAIF members](https://user-images.githubusercontent.com/3369400/133268513-5bfe2f93-4402-42c9-a403-81c9e86934b6.jpeg)
- ![Welcome Image with PLAIF members](https://raw.githubusercontent.com/plaif-dev/.github/main/assets/main/opening.png)
+ ![Welcome Image with PLAIF members](/assets/main/opening.png)
```

Reply:
Oh, I thought  relative path is not working in github. Thanks, I found the [reference](https://stackoverflow.com/a/36313391). Let me tag you again after editing.

Reply:
I change (not only) the image path to relative but also all file path in other files, too. Mind reviewing this again? 🙏